docs_src/microservices/device/profile/Ch-DeviceProfile.md

```diff
@@ -1,6 +1,93 @@
 # Device Profile
 
-![image](EdgeX_DeviceProfile.png)
+The device profile describes a type of device within the EdgeX system. Each
+device managed by a device service has an association with a device profile,
+which defines that device type in terms of the operations which it supports.
+
+For a full list of device profile fields and their required values see the [Device Profile Reference](./Ch-DeviceProfileRef.md)
+
+Identification
+--------------
+
+The profile contains various identification information. The `Name` field is required and must be unique in an EdgeX deployment. Other fields are optional and provide additional details:
+
+* Description
+* Manufacturer
+* Model
+* Labels
+
+DeviceResources
+---------------
+
+A deviceResource specifies a value within a device that may be read from or
+written to either individually or as part of a deviceCommand. It has a name
+for identification and a description for informational purposes.
+
+The device service allows access to deviceResources via its `device`
+REST endpoint.
+
+The `Attributes` in a deviceResource are the device-service-specific parameters
```

Comment:
In some of our original documentation we said:
Attributes are typically “inward facing”.  That is, attributes are used by the device service and provide details about how the device service can speak to the device to either get or set some of its values.  In other words, attributes are detail protocol and/or device specific information and inform the device service how to communication with the sensor to get (or set) the deviceResource (aka the value) of interest.